pkg/infra/httpclient/httpclientprovider/http_client_provider.go

```diff
@@ -23,6 +23,7 @@ func New(cfg *setting.Cfg, tracer tracing.Tracer) *sdkhttpclient.Provider {
 	middlewares := []sdkhttpclient.Middleware{
 		TracingMiddleware(logger, tracer),
 		DataSourceMetricsMiddleware(),
+		HTTPLoggerMiddleware(cfg.PluginSettings),
```

Comment:
should this be at the top or bottom of this list in order to capture any modifications to the request by other middleware?

Reply:
They run in order, so if the logger needs to see the last snapshot of the request, then it should be last